client/client.go

```diff
@@ -53,6 +55,7 @@ const (
 )
 
 type Client struct {
+	BaseURL         string
```

Comment:
I think it would be sufficient to make this an internal property?

Reply:
I just pushed a version that makes `Client.baseURL` internal.